plugins/aws/resoto_plugin_aws/resource/rds.py

```diff
@@ -463,6 +464,17 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
                 builder.dependant_node(
                     self, reverse=True, delete_same_as_default=True, clazz=AwsEc2Subnet, id=subnet_id
                 )
+        potential_keys = [
+            self.rds_kms_key_id,
+            self.rds_performance_insights_kms_key_id,
+            self.rds_activity_stream_kms_key_id,
+        ]
+        keys = [key for key in potential_keys if key]
+        for key_reference in keys:
+            if key_reference.startswith("arn:"):
+                builder.dependant_node(from_node=self, clazz=AwsKmsKey, arn=key_reference)
+            else:
+                builder.dependant_node(from_node=self, clazz=AwsKmsKey, id=key_reference)
```

Comment:
I remember commenting in another PR that we should have a method in AwsKmsKey for that kind of repetition. Do we have it somewhere?

Reply:
Oops this PR - am I missing something? I can not see it.

Reply:
@aquamatthias 
Ohh - I had misread that and thought you only meant the repetition we had here for the three Keys in RDS Instances 😅 
The ambivalence of Id or Arn for this edge (and a few others like it) doesn't lie within `AwsKmsKey` though, so I don't see how we could resolve it over there? It lies within the response of e.g. `rds describe-db-instances` 🤔

Reply:
@aquamatthias Here we go now!